app/views/posts/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="wsr">
+  <h1>New post</h1>
+  <% if @new_post.errors.any? %>
+    <div class="daa">
+      <% @new_post.errors.full_messages.each do |message| %>
+        <div> <%= message %></div>
+      <% end %>
+    </div>
+  <% end %>
+  <%= form_with model: @new_post, url: user_posts_path(@current_user)  do |form|  %>
+    <div class="wows">
+      <%= form.label :title ,class: "form" %>
+      <%= form.text_field :title ,class: "vcv" %>
+    </div>
+    <div class="wows">
+      <%= form.label :text ,class: "formlabel" %>
+      <%= form.text_area :text ,class: "area" %>
+    </div>
+    <%= form.submit "Add post" ,class: "btn" %>
+  <% end %>
+</section>
+<%# <%= form_for :post do |f| %> 
+  <%# <%= f.label :title %
+  <%# <%= f.text_field :title %>
+  <%# <%= f.label :body %> 
+  <%# <%= f.text_area :body %> 
+  <%# <%= f.submit "Create Post" %> 
+<%# <% end %> 
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧